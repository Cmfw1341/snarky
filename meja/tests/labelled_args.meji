let a : a:_ -> _;

let b : a:int -> int;

let c : ?a:bool -> unit -> bool;

let d : a:bool -> ?b:bool -> bool;

let e :
  unit -> (list(int), int, int, bool, bool, bool, bool, bool, bool, bool);

let f : a:int -> ?b:bool -> int -> int;

let g : a:int -> int -> int;

let h : a:int -> ?b:bool -> int;

let i : ?b:bool -> int -> int;

let j : unit -> int;

let j : a:{int} -> b:{option(int)} -> int;

let x : int;

let k : int;

let y : option(int);

let l : int;


