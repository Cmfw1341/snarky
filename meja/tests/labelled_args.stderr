File "tests/labelled_args.meja", line 39, characters 11-15:
Error: Unbound constructor true.
